package.json

```diff
@@ -10,7 +10,9 @@
 		"Other"
 	],
 	"activationEvents": [
-		"onStartupFinished"
+		"onStartupFinished",
+		"onCommand:Spotify.Pause",
```

Comment:
Also the commands listed here don't exist yet, since we've been using the `spotifymlh.` prefix for everything. I'm a fan of moving over to just `spotify.`, but that hasn't happened yet so these command activation events are extra not doing anything right now.

Reply:
Actually I'm going to do this over in a sec brb